.travis.yml

```diff
@@ -44,13 +44,22 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - #pip install codecov --user
+        - #pip install pytest-cov --user
+        - #pip install pytest-xdist --user
+        - #pip install kombu --user
+        - #pip install responses --user
```

Comment:
Remove these comments

Reply:
Sorry. I was testing codecov upload. Let me remove everything and create a fresh PR

Reply:
Actually I raised an issue for correcting the validation of the form. And was assigned new issue by @armenzg for running codecov for python tests. Looks like I forgot to create a new branch to work on it. So, I think I should rectify this particular PR and work on codecov on an independent branch. Apologies for the confusion @klahnakoski

Reply:
@shubhank-saxena   No problem, the point of review is also catch human error.  I have made the same mistake many times.